# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1310852251 14400
#      Sat Jul 16 17:37:31 2011 -0400
# Node ID a12569d5906edafd5582164046ef45cf32bf1947
# Parent  9f0f2c22605384a0b11217829281a6254365724d
codesprint: new tests for ispc, ismac, isunix

* ismac.m, ispc.m, isunix.m: Trivial new tests.

diff --git a/scripts/miscellaneous/ismac.m b/scripts/miscellaneous/ismac.m
--- a/scripts/miscellaneous/ismac.m
+++ b/scripts/miscellaneous/ismac.m
@@ -27,8 +27,10 @@ function retval = ismac ()
   if (nargin == 0)
     retval = octave_config_info ("mac");
   else
     print_usage ();
   endif
 
 endfunction
 
+%!error ismac (1);
+%!assert (islogical (ismac ()));
diff --git a/scripts/miscellaneous/ispc.m b/scripts/miscellaneous/ispc.m
--- a/scripts/miscellaneous/ispc.m
+++ b/scripts/miscellaneous/ispc.m
@@ -26,8 +26,11 @@ function retval = ispc ()
 
   if (nargin == 0)
     retval = octave_config_info ("windows");
   else
     print_usage ();
   endif
 
 endfunction
+
+%!error ispc (1);
+%!assert (islogical (ispc ()));
diff --git a/scripts/miscellaneous/isunix.m b/scripts/miscellaneous/isunix.m
--- a/scripts/miscellaneous/isunix.m
+++ b/scripts/miscellaneous/isunix.m
@@ -26,8 +26,11 @@ function retval = isunix ()
 
   if (nargin == 0)
     retval = octave_config_info ("unix");
   else
     print_usage ();
   endif
 
 endfunction
+
+%!error isunix (1);
+%!assert (islogical (isunix ()));
